src/twisted/protocols/ftp.py

```diff
@@ -2593,6 +2627,7 @@ def lineReceived(self, line):
         (Private) Parses the response messages from the FTP server.
         """
         # Add this line to the current response
+        line = line.decode(self._encoding)
```

Comment:
Ah I hadn't realised that'd perform differently, good spot.
Would it be it be less confusing to just say `if PYTHON2:` or is the convention to test for features instead?